buildSrc/build.gradle

```diff
@@ -0,0 +1,32 @@
+buildscript {
+    repositories {
+        jcenter()
+    }
+    dependencies {
+        classpath('org.eclipse.jgit:org.eclipse.jgit:5.10.0.202012080955-r')
```

Comment:
Dependency responsible for doing Git actions.

Reply:
such a peculiar naming strategy on their behalf! 🙃

Reply:
> Looking great so far - i've left a few nits and comments around.
> 
> I think most importantly we need some better documentation - i cannot seem to run the tests on my machine. I've tried Android Studio / IntelliJ / updating android studio? Am i missing something? How are you running the tests?
> 
> Additionally, I ran the task not expecting the changelog to change and found it updated it to this:
> 
> <img alt="Screenshot 2021-02-24 at 11 40 20" width="1204" src="https://user-images.githubusercontent.com/71286749/108995982-c313ee00-7695-11eb-962d-2051b739397c.png">
> 
> is this expected? I thought [this line](https://github.com/bitrise-steplib/bitrise-add-trace-sdk-android/pull/16/files#diff-593183f3e1a6d6b5f5cf6be2d56c136ec1566cc6307a1a9d58066495e816bafeR112) would be reached and no changes found meant cancelling the task? We also don't have a test for this scenario so this might be worth adding :)



> Looking great so far - i've left a few nits and comments around.
> 
> I think most importantly we need some better documentation - i cannot seem to run the tests on my machine. I've tried Android Studio / IntelliJ / updating android studio? Am i missing something? How are you running the tests?
> 
> Additionally, I ran the task not expecting the changelog to change and found it updated it to this:
> 
> <img alt="Screenshot 2021-02-24 at 11 40 20" width="1204" src="https://user-images.githubusercontent.com/71286749/108995982-c313ee00-7695-11eb-962d-2051b739397c.png">
> 
> is this expected? I thought [this line](https://github.com/bitrise-steplib/bitrise-add-trace-sdk-android/pull/16/files#diff-593183f3e1a6d6b5f5cf6be2d56c136ec1566cc6307a1a9d58066495e816bafeR112) would be reached and no changes found meant cancelling the task? We also don't have a test for this scenario so this might be worth adding :)

Discussed on Slack and in the call the testing issue, sorted it out, updated documentation, thanks for mentioning this. 👍 

Yes, it is intended to happen to have maintenance release. It is in the case, when there were commits, but none of the are in the allowed types (fix, feat, fix!, feat!). Not sure if we will ever do a release when we have only for example test, chore  commits, but in my opinion it looks better than a missing entry. Of course I am open to discussion, let me know your thoughts.
I added a test case for this, please find it here:
[https://github.com/bitrise-steplib/bitrise-add-trace-sdk-android/blob/APM-2391/buildSrc/src/test/java/io/bitrise/trace/internal/UpdateChangeLogTaskTest.java#L385](https://github.com/bitrise-steplib/bitrise-add-trace-sdk-android/blob/APM-2391/buildSrc/src/test/java/io/bitrise/trace/internal/UpdateChangeLogTaskTest.java#L385)

Reply:
Not sure why GH added this here, because I pushed on quote reply on the comment below 😕